### ISSUE TITLE
HMAC can't initialize

### ISSUE THREAD

**Issue by AndreasHeiberg** [iosdevzone/IDZSwiftCommonCrypto#13]
Tried from the docs:

```
let hmacs = HMAC(algorithm:.SHA256, key:"secret_key").update("content")?.final()
```

I get the error: "Type of expression is ambiguous without more context"

So I try:

```
let hmacs = HMAC(algorithm:HMAC.Algorithm.SHA256, key:"secret_key").update("content")?.final()
```

I get the error: "Cannot invoke initializer for type 'HMAC' with an argument list of type '(algorithm: HMAC.Algorithm, key: String)'"

**Comment by iosdevzone** [comment]
Whoops! Sorry about that. There was a missing `public` keyword in `HMAC.swift`. Should have a fix in about 10 minutes.

**Comment by iosdevzone** [comment]
Thanks for reporting this issue. It is solved in release 0.7.1. Sorry for any inconvenience!

**Comment by AndreasHeiberg** [comment]
Cool! Thanks for the quick fix :)
